values.yaml

```
@@ -40,6 +40,7 @@ global:
   # remove these checks to make the sync work.
   # If using bootstrapACLs then must be >= 0.10.1.
   # If using connect inject then must be >= 0.10.1.
+  # If using Consul namespaces, must be >= 0.12.
```

```suggestion
  # If using Consul Enterprise namespaces, must be >= 0.12.
```

In case people get confused?

---

Updated.